manifests/forward.pp

```diff
@@ -26,6 +26,7 @@
   $zone          = $name,
   $forward_first = 'no',
   $config_file   = $unbound::params::config_file,
+  $address_lookup = ''
```

Comment:
Can you please add a couple of doc lines regarding this newly introduced parameter?

Reply:
Done.